src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java

```diff
@@ -188,6 +189,15 @@ public String toString() {
     @FeatureToggle
     public boolean shiftClickForEquipment = false;
 
+    @Expose
+    @ConfigOption(name = "Dojo Rank Display", desc = "Display your rank, score and actual belt in the Challenges inventory.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean showDojoRankDisplay = false;
+
+    @Expose
+    public Position dojoRankDisplayPosition = new Position(100, 80, false, true);
```

Comment:
Can we move the default position to the right side of the inventory, so that is doesnt clash with the default location of the reputaion helper?